### PR TITLE
[Issue #8069] Fixed bug on description field

### PR DESCRIPTION
**Pull request by Maxco10**
Fixes issue https://github.com/firefly-iii/firefly-iii/issues/8069

Changes in this pull request:
- During the refactoring the description field was lost. Now, I update the RuleRepositories->store function with right code for description field.

@JC5

### CODE REVIEW

**Comment by JC5** [comment]
A rule does not need to have a description, could you make sure a similar check to the title is added?

**Comment by Maxco10** [comment]
@JC5 sorry, but i did not get your comment. Please read the issue.

File: app/Repositories/Rule/RuleRepository.php [hunk 1]
```diff
@@ -289,7 +289,7 @@ public function store(array $data): Rule
         $rule->strict          = array_key_exists('strict', $data) ? $data['strict'] : false;
         $rule->stop_processing = array_key_exists('stop_processing', $data) ? $data['stop_processing'] : false;
         $rule->title           = $data['title'];
-        $rule->description     = array_key_exists('stop_processing', $data) ? $data['stop_processing'] : null;
+        $rule->description     = $data['description'];
```
Comment (JC5) [inline app/Repositories/Rule/RuleRepository.php:292]: This field can be empty or non-existent. So I would expect this instead:

```
 array_key_exists('description', $data) ? $data['description'] : ''';
```

**Review by JC5 — CHANGES_REQUESTED** [review]

**Comment by Maxco10** [comment]
@JC5 PR Updated. I added check on title field, if the title does not exists, we set a default value "untitle", similar case for description field but the default value for this case is "" an empty string.

**Comment by JC5** [comment]
Could you fix the spelling of "Untitled" and use single quotes instead of double?

**Comment by Maxco10** [comment]
@JC5 Done!

**Pull request merged**